modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -95,10 +96,7 @@ def submit_form_to_central_mail
             status: #{status}, uuid #{confirmation_number}"
         )
 
-        json = { confirmation_number: }
-        json[:expiration_date] = 1.year.from_now if form_id == 'vba_21_0966'
-
-        render json:, status:
+        render json: get_json(confirmation_number, form_id), status:
```

Comment:
This bit of refactoring was necessary because the linter was complaining about this method being too long :(